init.lua

```diff
@@ -63,6 +63,14 @@ end)
 
 minetest.register_on_leaveplayer(function(player)
 	skins.ui_context[player:get_player_name()] = nil
+	player:get_inventory():set_size("hand", 0)
+end)
+
+minetest.register_on_shutdown(function()
+	local list = minetest.get_connected_players()
+	for _, player in pairs(list) do
```

Comment:
`get_connected_players()` is 100% guaranteed to be a continuous array, so you can safely use `ipairs` for the loop, which is faster.

Reply:
As you wish.

Reply:
What he meant was that `ipairs(minetest.get_connected_players())` could be used in this case. That's however less important for such small tables, so it's to me either way.